target/classes/META-INF/maven/com.spring.rest/BootApp/pom.properties

```diff
@@ -1,7 +1,7 @@
 #Generated by Maven Integration for Eclipse
-#Fri Sep 28 09:59:50 IST 2018
+#Tue Oct 09 00:13:58 IST 2018
 version=0.0.1-SNAPSHOT
 groupId=com.spring.rest
-m2e.projectName=BootApp
-m2e.projectLocation=D\:\\RestAPI\\BootApp
+m2e.projectName=java-springboot
+m2e.projectLocation=C\:\\Users\\Kritika\\git\\java-springboot

```

Comment:
Can we delete this Eclipse related file?
Or can we just put this file in .gitigonre?